Fix libtool error, and don't look for system pcap headers when we've
specified an alternate location.

https://github.com/aircrack-ng/aircrack-ng/pull/1848
--- ./build/m4/aircrack_ng_pcap.m4.orig	2018-04-03 15:00:23.000000000 -0500
+++ ./build/m4/aircrack_ng_pcap.m4	2018-04-03 23:12:19.000000000 -0500
@@ -43,7 +43,7 @@
         [use PCAP includes in DIR, [default=auto]])
     ],[
     	if test -d "$withval" ; then
-    		CPPFLAGS="$CPPFLAGS -I $withval"
+    		CPPFLAGS="$CPPFLAGS -I$withval"
     	fi
     ])
 
@@ -52,14 +52,14 @@
         [use PCAP libraries in DIR, [default=auto]])
     ],[
     	if test -d "$withval" ; then
-    		LDFLAGS="$LDFLAGS -L $withval"
+    		LDFLAGS="$LDFLAGS -L$withval"
     	fi
     ])
 
 dnl
 dnl Search for headers
 dnl
-if test "${with_libpcap_includes+set}" != set; then
+if test "${with_libpcap_include+set}" != set; then
 	AC_MSG_CHECKING(pcap header directories)
 
 	found_pcap_dir=""
